src/service.py

```diff
@@ -449,12 +450,34 @@ class DCGMExporter(SnapExporter):
 
     exporter_name: str = "dcgm"
     port: int = 9400
+    metrics_location: Path = Path("/var/snap/dcgm/common/")
```

Comment:
we can have a single variable for this:
```suggestion
    metrics_location: Path = Path("/var/snap/dcgm/common/dcgm-exporter-metrics-file")
```
If you need just the directory you can call `metrics_location.parent`

Reply:
Its the name of the snap configuration, not the metrics file